c10/core/SymIntNodeImpl.h

```diff
@@ -12,7 +12,6 @@ namespace c10 {
 
 class SymInt;
 class SymIntNodeImpl;
-using SymIntNode = c10::intrusive_ptr<SymIntNodeImpl>;
```

Comment:
You can duplicate a using multiple times. So there is no need to delete it from the other site.